rust/lancedb/src/remote/db.rs

```diff
@@ -54,6 +54,115 @@ impl ServerVersion {
     }
 }
 
+pub const OPT_REMOTE_PREFIX: &str = "remote_database_";
+pub const OPT_REMOTE_API_KEY: &str = "remote_database_api_key";
+pub const OPT_REMOTE_REGION: &str = "remote_database_region";
+pub const OPT_REMOTE_HOST_OVERRIDE: &str = "remote_database_host_override";
```

Comment:
use `OPT_REMOTE_PREFIX ` to connect "api_key", "region"?  Not sure if it's a good practise in RUST.

Reply:
Like this?
```suggestion
pub const OPT_REMOTE_API_KEY: &str = concat!(OPT_REMOTE_PREFIX, "key");
pub const OPT_REMOTE_REGION: &str = concat!(OPT_REMOTE_PREFIX, "region");
pub const OPT_REMOTE_HOST_OVERRIDE: &str = concat!(OPT_REMOTE_PREFIX, "host_override");
```